.github/scripts/test-info.mjs

```diff
@@ -185,6 +185,10 @@ const components = {
         certification: true,
         sourcePkg: ['configuration/redis', 'configuration/redis/internal'],
     },
+    'configuration.postgres': {
```

Comment:
Please run `make prettier-format`. A bit of a nit but this should go before `configuration.redis.v6` to keep alphabetic order :)